script.js

```diff
@@ -1,4 +1,72 @@
+
+// Project 1: scissors paper stone 
+// user input scissors or paper, or stone
+// input validation for "scissors" "paper" "stone"
+// assign 0,1,2 to scissors, paper, stone
+// random generate number 0,1,2
+// check input vs comPlay
+// winCondition: scissors > paper || paper > stone || stone > scissors 
+// drawCondition: input == computerRandom
+// updated for formatting 
+
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  input = input.toLowerCase();
+  var myOutputValue = "";
+  var input1 = "";
+
+//input validation
+if (input != "scissors" || 
+input != "paper" ||
+input != "stone") { 
+  myOutputValue = "Please try again, type 'scissors' 'paper' or 'stone'"
   return myOutputValue;
+}  
+  var comPlay = randomNumber()
+console.log (comPlay);
```

Comment:
Removing console.log before submitting is also standard practice. Something to remember for next time! :)